oct/cli/package/aws.py

```diff
@@ -30,25 +35,45 @@
     is_flag=True,
     help='Mark an AMI created previously as ready.',
 )
+@option(
+    '--tag',
+    '-t',
+    "tag",
```

Comment:
`tags` means you can pass multiple with one invocation. `tag` is strictly more correct. `env=prod` is one tag.